src/xcode/ENA/ENA/Source/HTTPClient/LocatorsNotYetUsed/SubmitPPA.swift

```diff
@@ -17,15 +17,24 @@ extension Locator {
 	) -> Locator {
 		let fake = String(isFake ? 1 : 0)
 		let forceApiHeader = String(forceApiTokenHeader ? 1 : 0)
+		
+		var defaultHeaders = [
+			"Content-Type": "application/x-protobuf",
+			"cwa-ppac-ios-accept-api-token": forceApiHeader,
+			"cwa-fake": fake
```

Comment:
I think before the refactoring, these were not set by default. Can you please double-check?

https://github.com/corona-warn-app/cwa-app-ios/compare/v2.24.1..v2.25.0-rc.5#diff-931f7490dd281c0b996106c696f4925213315ec8dbee5f55c7926af6d6054ab4

Reply:
sure

Reply:
for the second we do this for all the refactored requestsm for example check https://github.com/corona-warn-app/cwa-app-ios/blob/release/2.26.x/src/xcode/ENA/ENA/Source/HTTPClient/LocatorsNotYetUsed/KeySubmission.swift

So this was already being done before but directly in the session

Reply:
for cwa-ppac-ios-accept-api-token we have **#if !RELEASE** before and we have it now so we are sure we never add this in production

Reply:
for the **isFake** flag, what we normally did is to fire a request normally and then after we get the response we check the isFake flag, if it is set to true we return an error that it is a fakeRequest, this is for Plausible-deniability so it is done **also on production**,  here are some screenshot how we checked it before the new rest service and how we do it now after the refactoring:
Old HTTP Client:
<img width="844" alt="Screenshot 2022-07-27 at 09 34 17" src="https://user-images.githubusercontent.com/15270737/181193377-e1375465-790f-40ad-ad4a-34eef9b35a18.png">
New Rest Service:
<img width="1068" alt="Screenshot 2022-07-27 at 09 50 44" src="https://user-images.githubusercontent.com/15270737/181193475-ab407d50-2ea1-469b-8e56-63a9375d65d6.png">.
PS: We dont show a visual error to the user in this case